starter-code/src/clue.js

```diff
@@ -1,94 +1,122 @@
+    occupation:   "Actor"
+};
+var mrsPeacock = {
+    alias:        "mrsPeacock",
+    first_name:   "Eleanor",
+    last_name:    "Peacock",
+    color:        "blue",
+    description:  "She is from a wealthy family and uses her status and money to earn popularity",
+    age:          36,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation:   "Socialité"
+};
+var mrMustard = {
+    alias:        "mrMustard",
+    first_name:   "Jack",
+    last_name:    "Mustard",
+    color:        "yellow",
+    description:  "He is a former football player who tries to get by on his former glory",
+    age:          62,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation:   "Retired Football player"
+};
 
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+var rope = {name: "rope" ,weight: 10};
+var knife = {name: "knife",weight: 8};
+var candlestick = {name: "candlestick",weight: 2};
+var dumbbell = {name: "dumbbell",weight: 30};
+var poison = {name: "poison",weight: 2};
+var axe = {name: "axe",weight: 15};
+var bat = {name: "bat",weight: 13};
+var trophy = {name: "trophy",weight: 25};
+var pistol = {name: "pistol",weight: 20};
 
 // Rooms
-name: Dinning Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+var dinningRom = {name: "Dinning Room"};
+var conservatory = {name: "Conservatory"};
+var kitchen = {name: "Kitchen"};
+var study = {name: "Study"};
+var library = {name: "Library"};
+var billardRoom = {name: "Billiard Room"};
+var lounge = {name: "Lounge"};
+var ballroom = {name: "Ballroom"};
+var hall = {name: "Hall"};
+var spa = {name: "Spa"};
+var livingRoom = {name: "Living Room"};
+var observatory = {name: "Observatory"};
+var theater = {name: "Theater"};
+var guestHouse = {name: "Guest House"};
+var patio= {name: "Patio"};
+
 
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [mrGreen,drOrchid,profPlum,missScarlet,mrsPeacock,mrMustard];
 
 // Rooms' Collection
-var roomsArray = [];
+var roomsArray = [rope,knife,candlestick,dumbbell,poison,axe,bat,trophy,pistol];
 
 // Weapons Collection
-var weaponsArray = [];
+var weaponsArray = [dinningRom,conservatory,kitchen,study,library,billardRoom,lounge,ballroom,hall,spa,livingRoom,observatory,theater,guestHouse,patio];
+
+function randomSelector (array) {
+  var randomNumber = Math.floor(Math.random()*array.length);
+  var item = array[randomNumber];
```

Comment:
Os podéis ahorrar una de estas dos líneas poniendo directamente `return array[randomNumber]`